### PR TITLE
cfgSkeletons indentation formatting fixes

### PR DESCRIPTION
**Pull request by TheCandianVendingMachine**
**When merged this pull request will:**
- Change all instances of `class OFP2_ManSkeleton` to correct indentation of bone layout

### CODE REVIEW

File: addons/tacticalladder/data/model.cfg [hunk 1]
```diff
@@ -26,125 +26,118 @@ class CfgSkeletons {
...
+                                                "Face_EyelidUpperRight","Face_Hub",
+                                                "Face_EyelidUpperLeft","Face_Hub",
+                                                "Face_EyelidLowerRight","Face_Hub",
+                                                "Face_EyelidLowerLeft","Face_Hub",
+                                                "EyeLeft","Face_Hub",
+                                                "EyeRight","Face_Hub"
+        };// end of skeleton array
         // location of pivot points (local axes) for hierarchical animation
-        pivotsModel="A3\anims_f\data\skeleton\SkeletonPivots.p3d";
+        pivotsModel="Samples_F\TemplateRTM\ManSkeleton_Pivots.p3d";
```
Comment (dedmen) [inline addons/tacticalladder/data/model.cfg]: Changing pivots model. The old one was correct. Samples_F doesn't exist in normal play

Comment (dedmen) [reply addons/tacticalladder/data/model.cfg]: That is also wrong in the fastroping one and https://github.com/acemod/ACE3/pull/6773

Though afaik that entry isn't even used in-game, it's only needed for creation of rtm animations.

**Comment by jonpas** [comment]
We could symlink them, git supports that.

**Comment by dedmen** [comment]
>But there are small differences in each eh?

shouldn't. The pivot;Model is different, but that's because some use the samples one which will not be loaded in release version Arma. We should use the path that's always there.

**Comment by jokoho48** [comment]
that is wrong @dedmen they should be different in some places where models or Animations have a different behavior in that file or a model is referenced in the model.cfg

File: addons/tacticalladder/data/model.cfg [hunk 2]
```diff
@@ -26,125 +26,118 @@ class CfgSkeletons {
     class OFP2_ManSkeleton {
         isDiscrete = 0;
         skeletonInherit = "";
-        skeletonBones[] = {
+        skeletonBones[]= {
```
Comment (dedmen) [inline addons/tacticalladder/data/model.cfg]: whitespace before `=` should proably stay

**Comment by dedmen** [comment]
>they should be different in some places where models or Animations have different behavior in that file or a model is referenced in the model.cfg

Which is not the case here.
Or if that should be the case, it isn't. So I guess it doesn't matter.

One difference I just saw is that gunbag has some names lowercase that are uppercase in Arma 3 samples. Are bone names case sensitive or not? Cuz if they are then we have a problem, either the gunbag model.cfg is wrong, or Arma 3 samples is wrong.

File: addons/gunbag/data/model.cfg [hunk 3]
```diff
@@ -7,119 +7,118 @@ class CfgSkeletons {
...
+                                                "Face_Eyelids","Face_Hub",
+                                                "Face_EyelidUpperRight","Face_Hub",
+                                                "Face_EyelidUpperLeft","Face_Hub",
+                                                "Face_EyelidLowerRight","Face_Hub",
+                                                "Face_EyelidLowerLeft","Face_Hub",
+                                                "EyeLeft","Face_Hub",
+                                                "EyeRight","Face_Hub"
+        };// end of skeleton array
         // location of pivot points (local axes) for hierarchical animation
-        pivotsModel="A3\anims_f\data\skeleton\SkeletonPivots.p3d";
```
Comment (dedmen) [inline addons/gunbag/data/model.cfg]: Should stay the `A3\anims_f` path, for ever instance of the OFP2_ManSkeleton that we have

**Comment by dedmen** [comment]
@PabstMirror what do you think about the idea of putting the OFP skeleton into a single header file in ace common and just including that in the model.cfg? Would create a PR for that.

**Pull request merged**